tests/ha/hawk_gui.pm

```diff
@@ -83,7 +79,12 @@ sub run {
     assert_script_run "mkdir -m 1777 $path";
     assert_script_run "xhost +";
     barrier_wait("HAWK_GUI_CPU_TEST_START_$cluster_name");
-    my $docker_cmd = "docker run --rm --name test --ipc=host -v /tmp/.X11-unix:/tmp/.X11-unix -e DISPLAY=\$DISPLAY -v \$PWD/$path:/$path ";
+    # become root because podman command needs elevated permissions
+    # and then return to the user's home directory.
+    # If you delete this, also delete subsequent 'exit'
+    become_root;
+    assert_script_run("cd /home/bernhard");
```

Comment:
To avoid hard coding values, I would've gone here with:
```
assert_script_run "cd ~$testapi::username";
```
But this is a very minor thing. We can do it later :)